ROTATION.md

```diff
@@ -112,6 +112,73 @@ Modify the Kubernetes manifest
    conjur.org/secrets-refresh-enabled: "true"
    conjur.org/secrets-refresh-interval: 10m
    ```
+   [Here is the example push file manifest modified for rotation.](assets/p2f-rotation.yaml)
+
+   [For comparison, a push to to file manifest as an init container.](assets/push-to-file.yaml)
+
+   Secrets Rotation can also be used with Kubernetes secrets.
+
+   [Here is an example Kubernetes Secrets manifest modified for rotation.](assets/k8s-rotation.yaml)
+
+   [For comparison, a Kubernetes Secrets manifest as an init container.](assets/k8s-secrets.yaml)
+
```

Comment:
I like that you added example manifest files for the JWT authenticator. Do you think there's value in linking to them here along with the other ones?